### PR TITLE
[feature] Add Dockerfile and update readme

### PR DESCRIPTION
**Pull request by larrabee**
Hello. This pull request add dockerfile and install instructions to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,77 @@ If you are using the bundled ZoneMTA then you should make sure you are using a p
...
+      image: mailtrain-node:latest
+      container_name: "mailtrain-node"
+      links:
+        - "mailtrain-mysql:mailtrain-mysql"
+        - "mailtrain-redis:mailtrain-redis"
+      ports:
+        - "3000:3000"
+      volumes:
+        - "/etc/mailtrain/production.toml:/app/config/production.toml"
+  volumes:
```
Comment (witzig) [inline README.md]: What about GrapeJS and Mosaico upload folders?

Comment (larrabee) [inline README.md]: Thanks, did not know about it.

**Comment by andris9** [comment]
Dockerfile support has been requested before. Usually I have not been in favor of adding it as Dockerfiles can be composed in many ways and different users might prefer different setup (eg. by using a specific base image)

**Comment by larrabee** [comment]
Many users like install app by docker pull/build && copy-paste config && run. With default Dockerfile they can do it.  
Other user still able to build own docker image or change existing dockerfile.

**Comment by andris9** [comment]
OK, I guess it would make sense to add it. I think you should also install imagemagick, otherwise Mosaico image uploads would not work properly.

**Comment by witzig** [comment]
Yeah, a Dockerfile was the first thing I was looking for when discovering Mailtrain. Would be be great to have this.

**Comment by osixia** [comment]
Is there any reason to use centos as baseimage instead of more light weight baseimage like the official node:7.7.4-alpine for example ?

**Pull request merged**